packages/validate-config/package.json

```diff
@@ -18,7 +18,10 @@
   },
   "homepage": "https://github.com/oneblink/toolkit-js#readme",
   "devDependencies": {
-    "@types/lodash.isplainobject": "^4.0.7"
+    "@types/lodash.isplainobject": "^4.0.7",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/styled-components": "^5.1.25",
+    "client": "^0.0.1"
```

Comment:
why does a package used in client depend on it?

Reply:
hmmmm, these dependencies shouldn't be in validate-config.  I'll fix it up.